### ISSUE TITLE
Wrangler should be prescriptive about KV Namespace Titles

### ISSUE THREAD

**Issue by ashleymichal** [cloudflare/wrangler#551]
## The Problem
KV Namespaces are generally identified in the UI by their "title" property, a human-readable string which must be unique within an account. However, in the context of a Wrangler project, this property is generally only referenced when creating and listing namespaces. In all other cases, it makes more sense to refer to a namespace by its binding as specified in the project's wrangler.toml.

## The Solution
To ease confusion over the relationship between a title and a binding, Wrangler should provide an interface, feedback, and naming conventions that help to bridge these concepts.

### Example Interaction
``` sh
& wrangler kv create myblogassets --env=prod
Success! Namespace created: {
  title: "projectname-myblogassets", id: "2ce8545fc04147fab45aaceb9fb9393f"
}
Add the following to your wrangler.toml under `[env.prod]`:
kv-namespaces = [
  { binding: "myblogassets", id: "2ce8545fc04147fab45aaceb9fb9393f" }
]
```
Where the convention for namespace "title" is `<name>-<binding>`. This should preserve a reasonable level of uniqueness (worker names must be unique, but a user might reasonably use the same binding names across worker projects), while still providing human-readable namespace titles in the UI, and connecting a namespace title with its binding.

We can refine the help message based on things like whether they have existing kv namespaces ("add this to your kv-namespaces array"). Perhaps future iterations will just make a namespace per environment on your behalf as a convenience. We'll see.

We should update the KV command documentation accordingly to refer to this arg as `binding` rather than `title`.

EDIT: a previous conversation concluded that `binding + env` would be sufficient for uniqueness, but that only applies within a project. Updated to suggest we use `project name + binding` instead.

**Comment by gabbifish** [comment]
Question: do we even want to surface the title to users anymore? Or just give them the binding and id information, plain and simple?

```toml
$ wrangler kv create myblogassets --env=prod
Success! 
Add the following to your wrangler.toml under `[env.prod]`:
kv-namespaces = [
  { binding: "myblogassets", id: "2ce8545fc04147fab45aaceb9fb9393f" }
]
```

**Comment by EverlastingBugstopper** [comment]
I like this BUT I am worried that if we ask people to "Add the following to your `wrangler.toml`" when there are *already* namespaces there, we could get people doing things like

```toml
kv-namespaces = [
  { binding: "myblogassets", id: "2ce8545fc04147fab45aaceb9fb9393f" }
]
kv-namespaces = [
  { binding: "myotherassets", id: "f3939bf9becaa54baf74140cf5458ec2" }
]
```

**Comment by ashleymichal** [comment]
i have a solution to that...

**Comment by gabbifish** [comment]
closed by #557
